app/controllers/api/marketing/lists_controller.rb

```diff
@@ -0,0 +1,665 @@
+# 
```

Comment:
so before I start moving stuff around, there should be 2 files instead of one?...
- app/controllers/api/v1/marketing/lists_controller.rb
- lib/api/v1/marketing/Lists.class.rb

then the controller would contain only something like...
class Api::V1::Marketing::ListsController < ActionController::Base
    def get_list
        include Api::V1::Marketing::Lists
        Api::V1::Marketing::Lists.get_list
    end
end

?